File: Android-Demo-App/src/main/java/com/applovin/apps/demoapp/interstitials/InterstitialDemoMenuActivity.java
package com.applovin.apps.demoapp.interstitials;

import android.content.Intent;

import com.applovin.apps.demoapp.DemoMenuActivity;
import com.applovin.apps.demoapp.DemoMenuItem;

public class InterstitialDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Basic Integration", "Quick interstitial ads integration", new Intent( this, InterstitialBasicIntegrationActivity.class ) ),
                new DemoMenuItem( "Manually loading ad", "Use this for greater control over the ad load process", new Intent( this, InterstitialManualLoadingActivity.class ) ),
                new DemoMenuItem( "Zone Integration", "Create different user experiences of the same ad type", new Intent( this, InterstitialZoneActivity.class ) )
        };
        return result;
    }
}

File: AppLovin-Android-Kotlin-Demo-App/src/main/java/com/applovin/apps/kotlindemoapp/interstitials/InterstitialDemoMenuActivity.kt
package com.applovin.apps.kotlindemoapp.interstitials

import android.content.Intent
import com.applovin.apps.kotlindemoapp.DemoMenuActivity
import com.applovin.apps.kotlindemoapp.DemoMenuItem

class InterstitialDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Basic Integration", "Quick interstitial ads integration", Intent(this, InterstitialBasicIntegrationActivity::class.java)),
            DemoMenuItem("Manually loading ad", "Use this for greater control over the ad load process", Intent(this, InterstitialManualLoadingActivity::class.java)),
            DemoMenuItem("Zone Integration", "Create different user experiences of the same ad type", Intent(this, InterstitialZoneActivity::class.java))
    )
}
